packages/shorebird_cli/test/src/commands/release/release_ios_command_test.dart

```diff
@@ -17,6 +17,7 @@ import 'package:shorebird_cli/src/doctor.dart';
 import 'package:shorebird_cli/src/logger.dart';
 import 'package:shorebird_cli/src/os/operating_system_interface.dart';
 import 'package:shorebird_cli/src/platform.dart';
+import 'package:shorebird_cli/src/shorebird_build_mixin.dart';
```

Comment:
is this import needed?

Reply:
Yep - ExportMethod was moved to `shorebird_build_mixin`. I can move it to its own file if that makes more sense?